src/related-manager.ts

```diff
@@ -30,16 +29,17 @@ class RelatedManager extends KalturaPlayer.core.FakeEventTarget {
 
   private playByIndex(index: number) {
     this.isHiddenByUser = false;
-    if (this._areEntriesExternal) {
+    if (this.entryService.isPlayable(this.entries[index])) {
       this.plugin.player.setMedia({sources: this.entries[index]});
       this.plugin.player.play();
     } else {
       const entry = this.entries[index];
       this.plugin.player
         .loadMedia({...entry, entryId: entry.id, ks: this.ks})
```

Comment:
entry is a `Sources` so how it related to the loadMedia call?

Reply:
if an entry is missing source data and can't be played, we can get the needed data from loadMedia

Reply:
i asked about this `loadMedia({...entry,....` - the entry contains non relevant data for the loadMedia call